docs/guides/admin/context-links.md

```diff
@@ -0,0 +1,139 @@
+# Context Links
+
+This feature was introduced as part of [#2439](https://github.com/google/timesketch/issues/2439) and allows linking of specific event attributes and values to external lookup services for an easy analyst workflow.
+
+## Demo
+
+![TS_ContextLinks_RedirectWarn](https://user-images.githubusercontent.com/99879757/205897124-00154a89-e58a-4cf0-a89e-ba3b4f3052d8.gif)
```

Comment:
can we add this gif to the docs folder instead? https://github.com/google/timesketch/tree/master/docs/assets/images